prov/opx/configure.m4

```diff
@@ -50,6 +50,7 @@ AC_DEFUN([FI_OPX_CONFIGURE],[
 	AS_IF([test "x$macos" = "x1"],[opx_happy=0],
 		[test "x$freebsd" = "x1"],[opx_happy=0],
 		[test x$host_cpu != xx86_64],[opx_happy=0],
+		[test x$host_cpu != xriscv64],[opx_happy=0],
```

Comment:
This check is already implied by the previous line.

Reply:
Or do you mean `=` instead of `!=` ?

Reply:
@j-xiong thank you for double checking, looks like there's an issue on my end updating the autoconf scripts, cached versions are being run. Will fix this issue soon.